app.py

```diff
@@ -30,11 +29,67 @@ def send_mail(form_msg, form_name, form_email):
         # print("sent")
     except urllib.HTTPError:
         print("not sent")
+        slack_notifier(mode=1)
+    except Exception:
+        print("Some other exception occured. Not sent")
+        slack_notifier(mode=2)
+
+    from_email = Email(os.environ['KOSS_EMAIL'])
+    # print("email init")
+    to_email = Email(form_email)
+    # print("email init2")
+    subject = "Query Recieved"
+    # print("subject")
+    content = Content("text/plain", "Hi "+form_name+",\n\n"+"Your message has been recieved by us and we \
+                       will respond to it soon."+"\nThank-you for communicating with us, hope your query \
+                       will cleared by our team."+"\n\n\nKOSS IIT Kharagpur")
```

Comment:
I think its okay just to mention only "Thank-you for communicating with us" at the end. Because, it may not be a query rather a feedback.